src/transformers/quantizers/base.py

```diff
@@ -215,7 +215,10 @@ def dequantize(self, model):
 
         # Delete quantizer and quantization config
         del model.hf_quantizer
-
+        del model.config.quantization_config
+        del model.config._pre_quantization_dtype
+        model.is_quantized = False
```

Comment:
Quantizer seems better place for it than PreTrainedModel.

Not sure if that's 100% exhaustive, but it was enough to solve my issue.

Reply:
Nice, sounds good to me !